README.md

```diff
@@ -27,7 +27,18 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
 
 ## Installing the plugin into your IDE
 
-(coming soon)
+If you'd like to try out the plugin pre-release, you can build it from source and install it into your IDE:
+
+1) Clone this repository to your machine:<br />
+   `git clone git@github.com:GoogleContainerTools/google-container-tools-intellij.git`
+2) Build the plugin from the root of the repository:<br />
+   `./gradlew buildPlugin`
+3) Find the `.zip` binary in the root of the repository under: <br />
+   `build/distributions/google-container-tools-intellij.zip`
+4) Install the plugin into your JetBrains IDE:
+   1) Go to `Preferences > Plugins`
+   2) Select the settings gear icon and then `Install Plugin From Disk`
```

Comment:
What do you think about adding a single screenshot here where this menu is shown opened? It is kind of hidden in newer versions of IDEs and some users may not be familiar enough with this feature.